src/components/select/_select.scss

```diff
@@ -271,8 +271,6 @@
     position: absolute;
     right: 1rem;
     bottom: 1rem;
-    width: rem(10px);
```

Comment:
Did we want to change this to `width: 1rem; height: 1rem` to match the icon size? Can also add it in #1490 if we want

Reply:
@joshblack basically i was thinking that defining a width here would override whatever rules come from the icon-elements package… (

Reply:
basically i was thinking that defining a width here would override whatever rules come from the icon-elements package… if a width of 1em would not cause this issue, it might be a good fix!

Reply:
I don't think adding explicit width and height will help because what is actually 16x16 is the art board in which the svg lives in and not the svg itself. If we add those sizes it will be too big.